terminology/constants.bal

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# This is a map of implemented search params for CodeSystems.
+# These are defined as a map because to make the search process ease.
+public const map<string> CODESYSTEMS_SEARCH_PARAMS = {
+    _id: "id",
+    name: "name",
+    title: "title",
+    url: "url",
+    'version: "version",
```

Comment:
```suggestion
    version: "version",
```

Reply:
Which is the best practice **version** or **'version**? because some are suggested to use **'version**?

Reply:
We use `'` to escape keywords or identifiers starting with numbers eg:`'2a`. Since `version` is not a keyword there is no need to quote it

Reply:
if we remove the ' from version, it shows in a different color. That means color represent the keywords

Reply:
Checked but no color change for me.

Reply:
Check offline with Ballerina VSCode version 4.5.1, it seems a bug

Reply:
Updated in the code